src/webserver/WsBase.h

```diff
@@ -343,11 +344,10 @@ class WsSession : public WsBase,
 
             response = getDefaultWsResponse(id);
 
-            auto start = std::chrono::system_clock::now();
-            auto v = RPC::buildResponse(*context);
-            auto end = std::chrono::system_clock::now();
-            auto us = std::chrono::duration_cast<std::chrono::microseconds>(
-                end - start);
+            auto [v, timeDiff] =
+                util::timed([&]() { return RPC::buildResponse(*context); });
+
+            auto us = std::chrono::duration<int, std::milli>(timeDiff);
```

Comment:
Looks like it changed from microseconds to milliseconds here?

Reply:
The output of timed is milli , so the duration unit is milli.

Reply:
But `us` was in microseconds before, no? And also, why do we need to wrap `timeDiff` at all? it's already `millis` as it is returned from `timed` so i don't see why not just pass `timeDiff` instead of `us` if it's the same? 
In general, as long as the end result prints in the same unit as before - i don't mind how it's implemented too much :+1: